tests/puppeteer/pages/BO/BO_login.js

```diff
@@ -0,0 +1,31 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class BO_login extends CommonPage {
+
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Dashboard • PrestaShop';
+
+    this.emailInput = "#email";
+    this.passwordInput = "#passwd";
+    this.submitLoginButton = "#submit_login";
+    this.stayLoggedInCheckbox = "#stay_logged_in";
```

Comment:
Will need to update the selectors when the login page is migrated, because some of them will be changed

PR for refrence: https://github.com/PrestaShop/PrestaShop/pull/13733

Reply:
@rokaszygmantas can you please tell us what selectors will be changed and what's the new values of it ?
Thanks !

Reply:
@rokaszygmantas Are you sure these selectors need to be changed ? We need an ID to target inputs efficiently, and every selector change after 1760 will be labelled at critical bug and will need to be fixed ASAP.

Reply:
@SimonGrn it's not mandatory to change them during the login page migration, so if that brings any problems we can stick to the old ones :slightly_smiling_face:

Reply:
@rokaszygmantas if you can, that would be really nice for us and for maintenance to add IDs to every interesting element in the page: inputs, buttons, selectors, etc. That will make your job really easy and make the tests better.
Thanks !